tests/unit/Exports/OrderExportTest.php

```diff
@@ -61,14 +61,19 @@ public function assertNewOrder($new_order_id, $sk_customer_id, $new_order, $sk_o
         $this->assertEquals(1, $shipping['quantity'], 'quantity');
         $this->assertEquals(10, $shipping['ppu_wt'], 'ppu_wt');
 
+        $expectedBillingStreet = $new_order['billing_address_1'].' '.$new_order['billing_address_2'];
+        if ('NL' === $wc_order->get_billing_country()) {
+            $expectedBillingStreet = $new_order['billing_address_house_number'].' '.$new_order['billing_address_1'].' '.$new_order['billing_address_2'];
+        }
+
         $expect_billing = [
             'name' => $new_order['billing_first_name'].' '.$new_order['billing_last_name'],
             'isprivate' => empty($new_order['billing_company']),
             'address_billing' => [
                     'state' => $new_order['billing_state'],
                     'city' => $new_order['billing_city'],
                     'zipcode' => $new_order['billing_postcode'],
-                    'street' => $new_order['billing_address_1'].' '.$new_order['billing_address_2'],
+                    'street' => $expectedBillingStreet,
```

Comment:
use `streetnumber` for the 
```suggestion
                    'street' => $new_order['billing_address_1'].' '.$new_order['billing_address_2'],
                    'streetnumber' => $new_order['billing_address_house_number'],
```

Reply:
Initial test is wrong as well upon checking.
Now fixed.